src/config/ci.toml

```diff
@@ -1,5 +1,5 @@
 [meta]
-disable_recovery = true
+disable_recovery = false
```

Comment:
Enable `RECOVER` use in tests.

Reply:
I think we can make it finer-grained by only allowing ad-hoc recovery while still rejecting failure recoveries. This is to avoid false positives in tests.

Reply:
Fixed